.github/workflows/sub_lint.yml

```diff
@@ -77,7 +77,73 @@ on:
         type: boolean
         required: false
       cpplintFilters:
-        default: -whitespace-*
+        default: >-
```

Comment:
Is there some documentation available of what all these checks do?

Reply:
cpplint was originally Google's internal tool for enforcing their coding style:
https://google.github.io/styleguide/cppguide.html

The actual documentation for cpplint is a bit thin. I could easily be persuaded that we shouldn't be running cpplint at all -- clang-tidy and clazy do a pretty good job of catching the things we care about, and I'd bet that clang-tidy has some of these cpplint checks in it should we care to enable them.